tests/eclient/testdata/acl.txt

```diff
@@ -28,13 +30,13 @@ exec -t 10m bash dns_lookup.sh zededa.com
 source .env
 
 # Create network for which ACLs will be defined.
-eden network create 10.11.12.0/24 -n n1 -s {{$fake_domain}}:$host_ip
-test eden.network.test -test.v -timewait 10m ACTIVATED n1
+eden network create 10.11.12.0/24 -n {{$network_name}} -s {{$fake_domain}}:$host_ip
+test eden.network.test -test.v -timewait 10m ACTIVATED {{$network_name}}
 
 # First app is only allowed to access github.com and $long_domain.
-eden pod deploy -n curl-acl1 --memory=512MB docker://itmoeve/eclient:0.4 -p 2223:22 --networks=n1 --acl=n1:github.com --acl=n1:{{$long_domain}}
+eden pod deploy -n curl-acl1 --memory=512MB docker://itmoeve/eclient:0.4 -p 2223:22 --networks={{$network_name}} --acl=n1:github.com --acl=n1:{{$long_domain}}
```

Comment:
`--acl=n1:...` can be now replaced with `--acl={{$network_name}}:...`

Reply:
Good catch, thank you!